types/ember/tsconfig.json

```diff
@@ -16,6 +16,7 @@
         },
         "types": [],
         "noEmit": true,
+        "exactOptionalPropertyTypes": true,
```

Comment:
While this implicitly will ensure that all the `@types/ember__*` packages that `@types/ember` itself pulls in are compatible with `exactOptionalPropertyTypes`, anyone who's working on those types won't get feedback that they've done something incompatible until the full test suite runs.

Enabling the flag on those packages individually will give in-editor feedback right away if someone does something `exactOptionalPropertyTypes` doesn't like. It will also help down the line if and when it becomes more common _only_ to use those packages without `@types/ember` itself.

Given that, I think we should probably flip this flag across all the `types/ember__*` packages together.

Reply:
Sure, will do! Only for the "v4" version, right?

Reply:
Yep, I wouldn't worry about the older versions 👍

Reply:
Done! I also added it to `htmlbars-inline-precompile` and `rsvp`, not sure if I had to?